lib/src/shared/main/java/com/couchbase/lite/internal/support/Log.java

```diff
@@ -349,6 +352,14 @@ public static void setC4LogLevel(@NonNull EnumSet<LogDomain> domains, @NonNull L
         }
     }
 
+    public static void warn() {
+        if (WARNED.getAndSet(true)) { return; }
+        Log.w(
+            LogDomain.DATABASE,
+            "Database.log.getFile().getConfig() is now null: logging is disabled.  "
```

Comment:
We may need to change this message a little bit as the message will also be shown when setting log level to None.

Reply:
I agree...   I didn't want to differ from the other platform messages, but we might want to revisit this.